src/internal/hooks/useMessageReceivers.ts

```diff
@@ -63,9 +63,7 @@ export const useCommonMessageReceivers = (
 
   useReceiveMessage("getThemeData", TARGET_ORIGINS, (send, payload) => {
     const payloadString = payload as unknown as string;
-    const themeData = payloadString
-      ? jsonFromEscapedJsonString(payloadString)
-      : {};
+    const themeData = payloadString ? JSON.parse(payloadString) : {};
```

Comment:
Are you sure we don't need the escaped function? This comes from the DB right?

Reply:
Was able to use single quotes for the font names, which also fixed the issue, so we can just leave this as it is